<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>800</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>800</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>500</width>
       <height>600</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>1000</width>
       <height>1000</height>
      </size>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>531</width>
        <height>511</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetFixedSize</enum>
       </property>
      </layout>
     </widget>
     <widget class="QWidget" name="widget" native="true">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>-20</y>
        <width>221</width>
        <height>701</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>500</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>600</width>
        <height>900</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="btn_create">
         <property name="minimumSize">
          <size>
           <width>120</width>
           <height>60</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>120</width>
           <height>60</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>华文琥珀</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="mouseTracking">
          <bool>false</bool>
         </property>
         <property name="toolTip">
          <string>开始一个新的游戏</string>
         </property>
         <property name="text">
          <string>新游戏</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_help">
         <property name="minimumSize">
          <size>
           <width>120</width>
           <height>60</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>120</width>
           <height>60</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>华文琥珀</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="toolTip">
          <string>检查当前所填数字是否有误</string>
         </property>
         <property name="text">
          <string>检查</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_change">
         <property name="minimumSize">
          <size>
           <width>120</width>
           <height>60</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>120</width>
           <height>60</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>华文琥珀</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>更换难度</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_answer">
         <property name="minimumSize">
          <size>
           <width>120</width>
           <height>60</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>120</width>
           <height>60</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>华文琥珀</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>ArrowCursor</cursorShape>
         </property>
         <property name="toolTip">
          <string>查看当前数独的答案</string>
         </property>
         <property name="text">
          <string>查看答案</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_replay">
         <property name="minimumSize">
          <size>
           <width>120</width>
           <height>60</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>120</width>
           <height>60</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>华文琥珀</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="toolTip">
          <string>重玩当前游戏</string>
         </property>
         <property name="text">
          <string>重玩</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_submit">
         <property name="minimumSize">
          <size>
           <width>120</width>
           <height>60</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>120</width>
           <height>60</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>华文琥珀</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="toolTip">
          <string>提交检查答案</string>
         </property>
         <property name="text">
          <string>提交答案</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
